docs/book/features/integrations.md

```diff
@@ -23,6 +23,7 @@ These are the third-party integrations that ZenML currently supports:
 | ----------- | ------ | ---- | -------------------- | ------- |
 | Apache Airflow | ✅ | Orchestrator | Works for local environment | [airflow_local](https://github.com/zenml-io/zenml/tree/main/examples/airflow_local) |
 | Apache Beam | ✅ | Distributed Processing |   |   |
+| AWS | ✅ | Cloud |   |   |
```

Comment:
Maybe add a bit more detail here? :)

Reply:
We've really only added the ability to hook into S3 buckets for artifact stores and their filesystem, so we should be specific about that, I guess.

Reply:
True, I was lazy and just copied the GCP line :D
I'll add a little description

Reply:
done